packages/slice-machine/src/features/slices/sliceCards/SharedSliceSelectionCard.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Checkbox, Text } from "@prismicio/editor-ui";
+import type { FC } from "react";
+
+import { ComponentUI } from "@lib/models/common/ComponentUI";
+import { Card, CardFooter, CardMedia } from "@src/components/Card";
+import { getScreenshotUrl, getSharedSliceLibraryName } from "@src/domain/Slice";
+
+type SharedSliceSelectionCardProps = {
+  isComingSoon: boolean;
+  isSliceTemplate: boolean;
+  onSelectedChange: (selected: boolean) => void;
+  selected: boolean;
+  slice: ComponentUI;
+};
+
+export const SharedSliceSelectionCard: FC<SharedSliceSelectionCardProps> = ({
+  isComingSoon,
+  isSliceTemplate,
+  onSelectedChange,
+  selected,
+  slice,
+}) => {
+  const defaultVariation = ComponentUI.variation(slice);
```

Comment:
⚠️ if `defaultVariation` is `undefined`. you don't opt-out like before by returning `null` is it something that was wanted before? Or it was a mistake